themes/edx.org/lms/templates/header.html

```diff
@@ -1,5 +1,5 @@
 ## mako
-<%page expression_filter="h" args="online_help_token"/>
+<%page expression_filter="h" args="online_help_token,doc_base_url"/>
```

Comment:
When I tested on a sandbox, I was very confused (for about an hour) why the help link was not sending me to the correct location. I finally discovered this "edx.org" theme file, which isn't used on devstack or by tests. I'm sure this **never** results in bugs in production.